tmk_core/common/chibios/eeprom_teensy.c

```diff
@@ -517,7 +517,7 @@ void eeprom_write_block(const void *buf, void *addr, uint32_t len) {
 #else
 // No EEPROM supported, so emulate it
 
-#    define EEPROM_SIZE 32
+#    define EEPROM_SIZE 64
```

Comment:
NOTE: Increased this because eeconfig has already gone past 32.

Reply:
This may not work.   

I believe that this was set to 32 because of the KINETIS boards, which may be stuck with the size.  However, I'm no familiar enough with those controllers to be certain. 

Sorry for the tags, but I want to make sure here.
@fredizzimo @jackhumbert @skullydazed

Reply:
That change is only applicable for the line below -- a static buffer in RAM, not EEPROM.
The kinetis code hasn't been modified in that respect, was purely just to make sure we've got the extra space currently being used by the newer eeconfig entries.

Reply:
Unfortunately line 53 hasn't been modified, didn't touch it for similar reasons to your query.
Unsure how to handle this, to be honest.

Reply:
Tested this out on an ergodox infinity.  Can confirm 100% that it does not work.   Using CG_SWAP to test, as this uses the 33 byte to store the settings. Default layer changes are retained, but CG_SWAP is not. 

So, yeah, these boards are stuck at 32 bytes. :(

Reply:
This specific instance is still okay, given that it's a buffer in RAM. The first instance on line 54 is the real issue -- once the EEPROM is configured, its size is locked. I've rolled it back to 32 bytes for the EEPROM configuration on teensy, but added an override if people are using it on brand-new chips. Warning added.